designs/virtualization/4.x/cd roms/cd roms.md

```diff
@@ -0,0 +1,39 @@
+---
+parent: Virtualization
+version: 4.x
+---
+# CD ROMs
+
+## CD-ROMs on VM Details page
+
+It's been determined that separating CD-ROMs from the rest of the disks has technical overhead and complicates the flow. It makes more sense to include them as a lower level item in the disks rather than highlight them in the details view like we currently do.
+
+![CDROMs are on the details page](img/Details.png)
+
+We'll add a ‘Drive’ column to the ‘Disks’ table view.
+The default type will be ‘Disk’ and the other option(s) will be ‘CD ROM’.
+
+![adding a new Drive column](img/Type-column-added-table-view.png)
+
+## CD-ROMs on Create VM wizard
+
+We'll remove the CD ROMs from the ‘Advanced’ step and place it in the Storage step.
+For cases where we point the user to them (windows guest tools), we’ll direct the users to the Disks table under the Storage step.
+
+![notification directing to the disks table](img/Notification-to-disks-table.png)
```

Comment:
Should we update the text to say :
" see Disk-name in the storage step for more details"
We could link the disk name to the modal and they could look at it in the general step if they really wanted to.

Reply:
Thanks @matthewcarleton +1